keyRegistration/js/registration.js

```diff
@@ -63,10 +62,11 @@ $(document).ready(function()
         },
         error: function (xhr, ajaxOptions, thrownError) {
             setError(thrownError);
-       },
+        },
+
     });
 
-    function setError(text){
+    function setError(text) {
```

Comment:
Opening brace should be on a new line

*PHPCodeSnifferBear, severity NORMAL, section `js`.*